lib/game/entities/enemy.dart

```diff
@@ -82,7 +81,6 @@ class Enemy extends PhysicalEntity<SuperDashGame> {
         SpriteAnimationComponent(
           size: Vector2.all(gameRef.tileSize),
           animation: bettleAnimation,
-          anchor: Anchor.center,
```

Comment:
need to get rid of the anchor center, as latest leap needs it to be TopLeft (not sure why).

this is defaulted to TopLeft, if you remove the anchor